### PR TITLE
Kafka role improvements

### PR DESCRIPTION
**Pull request by mandoz**
1. Improved management of vars
2. Allow defining zookeeper chroot for kafka
3. Place kafka in `/opt` as recommended instead of using the home directory.
4. Small improvements.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -1,16 +1,23 @@
 ---
 
+kafka_path                        : "/opt/kafka_{{ kafka_scala_version }}-{{ kafka_version }}"
+kafka_hosts                       : "{{ groups['bi-kafka'] }}"
```
Comment (endeveit) [inline defaults/main.yml]: I think it's better to leave this as an empty list "[]".
To not rely on our internal infra stuff.

File: defaults/main.yml [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
+kafka_zookeeper_hosts             : "{{ groups['bi-zookeeper'] }}"
```
Comment (endeveit) [inline defaults/main.yml]: Same for zookeper stuff

**Comment by endeveit** [comment]
Looks good, but I think it's better not to hardcode our own stuff.
We can redeclare any variable in our group vars.

**Comment by mandoz** [comment]
Yes of course I meant to leave it generic. Silly copy paste mistake :) ill update the PR and let you know

**Comment by mandoz** [comment]
@endeveit branch updated.

**Comment by endeveit** [comment]
👍

**Pull request merged**